### DISCUSSION TITLE
How can I set the default window size of a plot window in linux?

### DISCUSSION THREAD

**Discussion by brandy125** [gnudatalanguage/gdl#1733] [Q&A]
Is there a way to set the default window size of a plot window in linux?

**Comment by GillesDuvert** [comment]
normally, setting env. var. GDL_GR_X_WIDTH and GDL_GR_X_HEIGHT should do it.

**Comment by alaingdl** [comment]
Weird ! On my Linux laptop, GDL_GR_X_WIDTH is working fine but GDL_GR_Y_WIDTH has no effect !
(too late to explore)

**Comment by GillesDuvert** [comment]
Indeed, GDL_GR_Y_WIDTH is not tested. sorry

**Comment by alaingdl** [comment]
Bad idea to test too late ! (stupid question !)

All the useful variables are prefixed by GDL_GR_X_
https://www.irya.unam.mx/computo/sites/manuales/IDL/Content/Creating%20IDL%20Programs/Appendices/prefs_gr_x.html

Do we have to consider to check the IDL ones ?  (with the same logic than IDL_PATH/GDL_PATH ?) Is it useful ? Request ?

**Comment by brandy125** [comment]
Thanks @alaingdl and @GillesDuvert!

Tthe two variables

GDL_GR_X_WIDTH 

and

GDL_GR_X_HEIGHT

did the trick  for me!

**Comment by GillesDuvert** [comment]
@brandy125 thanks for the test, indeed I programmed  GDL_GR_X_HEIGHT and that is what the documentation says.
@alaingdl Unfortunately, I changed the code to GDL_GR_Y_HEIGHT using the GH editor in the meantime, so will have to revert the change.
